### DISCUSSION TITLE
Startup Failed checks won't show

### DISCUSSION THREAD

**Discussion by Pantelis-Santorinios** [community/community#16152] [Actions]
What we see:

Workflows that fail with `startup failure` will not show in pull request checks.
If those checks are set as required then those are getting "stuck" with the message:

>Expected - Waiting for status to be reported

1.  Create a workflow that includes an external action like `actions/checkout`, and gets triggered on `pull_request`
2.  Change your Actions policy to Allow local actions only
3.  Make a pull request
4.  Additionally you may want to add branch protection

Expected behavior:

When the workflow fails to start with the `startup failure` error the associated check runs should fail and be seen as failed in the pull request.

**Comment by ghost** [comment]
Any updates from the GitHub support team on this?

**Reply by dave2wave** [reply]
This trouble makes scanning dependabot PRs more time consuming. `startup failure` should be marked in red and not black and the failed workflow should be included in the Check results in the conversation.

<img width="934" height="551" alt="Screen Shot 2026-01-30 at 8 39 41 AM" src="https://github.com/user-attachments/assets/16aaed48-a778-472f-81cf-5c4fc1e1255a" />
<img width="932" height="472" alt="Screen Shot 2026-01-30 at 9 11 21 AM" src="https://github.com/user-attachments/assets/781a5b7d-ea1e-4c53-8aa8-cc6eb8f135e2" />

Thanks!
